lib/puppet/type/dism.rb

```diff
@@ -1,19 +1,7 @@
 Puppet::Type.newtype(:dism) do
   @doc = 'Manages Windows features via dism.'
 
-  ensurable do
-    desc 'Windows feature install state.'
-
-    defaultvalues
-
-    newvalue(:present) do
-      provider.create
-    end
-
-    newvalue(:absent) do
-      provider.destroy
-    end
-  end
+  ensurable
```

Comment:
Sounds good, as these were previously not the case when this module was first developed.